docs/single-to-multi-master.md

```diff
@@ -0,0 +1,213 @@
+# Migrating from single to multi-master
+
+This document describes how to go from a single-master cluster (created by kops)
+to a multi-master cluster.
+
+## Warnings
+
+This is a risky procedure that **can lead to data-loss** in the etcd cluster.
+Please follow all the backup steps before attempting it. Please read the
+[etcd admin guide](https://github.com/coreos/etcd/blob/v2.2.1/Documentation/admin_guide.md)
+before attempting it.
+
+During this procedure, you will experience **downtime** on the API server, but
+not on the end user services.
```

Comment:
Perhaps add another line giving some indication of what will work and won't work (e.g. you can't create new pods, but existing pods will continue to respond to requests).